### DISCUSSION TITLE
Moving a part of diagram is not working properly for Edited edges points

### DISCUSSION THREAD

**Discussion by bogy0** [xyflow/xyflow#4488] [Ideas]
### What platform were you using when you found the bug?

- React Flow latest
- Browser and version:  Chrome
- OS and version: Mac OSX Sonoma 14.4.1

### Live code example

It can be reproducible on the demo page: https://reactflow.dev/examples/edges/editable-edge

### Describe the Bug

Editable edge points are not moving with their nodes.

### Steps to reproduce the bug or issue

1. Use Editable Edges feature and create some edges pointers between two nodes.
2. Now Select the two nodes and try to move around the nodes.
3. You will see, that edge points stay place and not affected by the reposition of their nodes.

### Expected behavior

I would expect that the editable edge route moves relatively with their nodes.

### Screenshots or Videos
![editable-edge-issue-when-moving-selected-nodes](https://github.com/user-attachments/assets/41a258d4-7e42-4913-ad36-a0ab5b1b3195)

For crossreference i posted this into Discord also: https://discord.com/channels/771389069270712320/1265693961935126612/1265693961935126612

**Comment by moklick** [comment]
In the current implementation you can't select the control points with a selection box. I will discuss with the team if we can add it!

**Reply by TruDan** [reply]
Do you have an update on that discussion with the team? This is a feature that would be really great to have.

Or at least selectable elements that aren't nessecarily 'nodes' - in our case we are building a diagram editor which is perhaps stretching the scope of xyflow's purpose, but this would save a lot of hacky overrides if it were built in.

For anyone finding this issue in the future, i managed to get it working using the onSelectionDrag event, and comparing the deltaT of the nodes, then offsetting the points in the edge. Don't want to paste my code here since its a pro example, but its really simple to workaround this specific issue
